regression/cbmc/struct15/test.desc

```diff
@@ -0,0 +1,11 @@
+CORE
+main.c
+--trace --smt2
```

Comment:
I don't think this code is even exercised without `--z3`, see https://github.com/diffblue/cbmc/blob/develop/src/solvers/smt2/smt2_conv.cpp#L101

Reply:
If `--smt2` is specified and a specific solver is not specified, then z3 is selected by default. See https://github.com/diffblue/cbmc/blob/develop/src/cbmc/cbmc_parse_options.cpp#L428 and https://github.com/diffblue/cbmc/blob/develop/src/cbmc/cbmc_parse_options.cpp#L438
That said, I think this would be a more robust test if `--z3` was specified, rather than relying on the default.

Reply:
Thanks, that's a good point! I will make this change.